### DISCUSSION TITLE
Can anyone help me here with all the debug warnings and errors?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#6127] [Support]
I have 3 full node running in the same network all with private keys. I have NOT opened the 8444 port because it is unsafe.

I have disabled upnp on 2 full nodes and on 1 is enabled. My modem has upnp DISABLED for security reasons obviously.

Main question is, shouldnt it be ok for all 3 nodes as it is to farm without errors?

I am farming 1 months and havent got a coin and its probably something wrong with the nodes.
I am getting errors and warnigns about connections etc and I cannot know if the full node is actually working fine. (GUI seems ok but it seems that it does not)

on the node with upnp enabled i get 
WARNING  Banning xx.xx.xx.x for 600 seconds...

on the other node:
[WinError 10054] An existing connection was forcibly closed by the remote host...

and on another:
INFO     0 plots were eligible for farming.....

What is going on, this is getting out of hand if you want to farm with more than 1 node.
Is it ok to have 3 full nodes on the same private network?

**Comment by BrandtH22** [comment]
@GeorgePapageorgakis , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - detailed instructions for farming on multiple machines can be found here https://docs.chia.net/farming-on-many-machines

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
